source/WesternStatesWater.WaDE.Managers.Api/Mapping/OgcApiProfile.cs

```
@@ -20,5 +23,31 @@ public OgcApiProfile()
             Contracts.Api.Responses.V2.CollectionMetadataGetResponse>();
         CreateMap<Engines.Contracts.Ogc.Responses.CollectionsResponse,
             Contracts.Api.Responses.V2.CollectionsMetadataGetResponse>();
+
+        CreateMap<Contracts.Api.Requests.V2.SiteFeaturesSearchRequest,
+                Engines.Contracts.Ogc.Requests.SiteFeaturesRequest>()
+            .ForMember(dest => dest.BoundingBox, mem => mem.MapFrom(src => src.Bbox))
+            .ForMember(dest => dest.LastSiteUuid, mem => mem.MapFrom(src => src.Next));
+
+        CreateMap<Engines.Contracts.Ogc.Responses.SiteFeaturesResponse,
+            Contracts.Api.Responses.V2.SiteFeaturesSearchResponse>();
+
+        CreateMap<Contracts.Api.Requests.V2.FeaturesSearchRequestBase,
+            Engines.Contracts.Ogc.Requests.FeaturesRequestBase>();
+    }
+}
+
+// This method is assuming Validators are in place to ensure the string is in the correct format.
+public class StringToBoundingBoxConverter : ITypeConverter<string, double[][]>
+{
+    public double[][] Convert(string source, double[][] destination, ResolutionContext context)
+    {
+        if (source == null)
```

If source cannot be null here (represents an error on our part), then I'd think about throwing an `InvalidOperationException` rather than returning null

---

Bbox is an optional parameter so source can be null, in which case we want to return null. This assembly doens't have nullables enabled.